src/AppBundle/Entity/Report/Report.php

```diff
@@ -233,12 +233,21 @@ class Report
      */
     private $metadata;
 
+
+    /**
+     * @JMS\Type("AppBundle\Entity\Report\Document")
+     * @var Document[]
```

Comment:
* @JMS\Type("ArrayCollection<AppBundle\Entity\Document>")
I think Doctrine needs this to be an ArrayCollection in order to keep track of associations and to help lazy loading